benchmarks/dusk_benchmarks.rs

```diff
@@ -85,102 +91,138 @@ mod scalar_benches {
     use zerocaf::field::FieldElement;
     use zerocaf::scalar::Scalar;
 
-
     /// `C = 182687704666362864775460604089535377456991567872`.
     pub static C: Scalar = Scalar([0, 0, 0, 2, 0]);
 
     /// `D = 904625697166532776746648320197686575422163851717637391703244652875051672039`
-    pub static D: Scalar = Scalar([2766226127823335, 4237835465749098, 4503599626623787, 4503599627370493, 2199023255551]);
+    pub static D: Scalar = Scalar([
+        2766226127823335,
+        4237835465749098,
+        4503599626623787,
+        4503599627370493,
+        2199023255551,
+    ]);
 
     pub fn bench_scalar_element_ops(c: &mut Criterion) {
-        c.bench(
-            "Scalar",
-            Benchmark::new("Addition", |b| b.iter(|| &C + &D))
-        );
+        c.bench("Scalar", Benchmark::new("Addition", |b| b.iter(|| &C + &D)));
 
         c.bench(
             "Scalar",
-            Benchmark::new("Subtraction", |b| b.iter(|| &C - &D))
+            Benchmark::new("Subtraction", |b| b.iter(|| &C - &D)),
         );
 
-        c.bench(
-            "Scalar",
-            Benchmark::new("Mul", |b| b.iter(|| &C * &D))
-        );
+        c.bench("Scalar", Benchmark::new("Mul", |b| b.iter(|| &C * &D)));
 
         c.bench(
             "Scalar",
-            Benchmark::new("Squaring", |b| b.iter(|| C.square()))
+            Benchmark::new("Squaring", |b| b.iter(|| C.square())),
         );
 
-        c.bench(
-            "Scalar",
-            Benchmark::new("Half", |b| b.iter(|| C.half()))
-        );
+        c.bench("Scalar", Benchmark::new("Half", |b| b.iter(|| C.half())));
     }
 }
 
 mod edwards_benches {
 
     use super::*;
-    use zerocaf::edwards::{EdwardsPoint, ProjectivePoint, CompressedEdwardsY};
-    use zerocaf::scalar::Scalar;
-    use zerocaf::field::FieldElement;
     use subtle::Choice;
+    use zerocaf::edwards::{CompressedEdwardsY, EdwardsPoint, ProjectivePoint};
+    use zerocaf::field::FieldElement;
+    use zerocaf::scalar::Scalar;
 
     pub static P1_EXTENDED: EdwardsPoint = EdwardsPoint {
         X: FieldElement([23, 0, 0, 0, 0]),
-        Y: FieldElement([1664892896009688, 132583819244870, 812547420185263, 637811013879057, 13284180325998]),
+        Y: FieldElement([
+            1664892896009688,
+            132583819244870,
+            812547420185263,
+            637811013879057,
+            13284180325998,
+        ]),
         Z: FieldElement([1, 0, 0, 0, 0]),
-        T: FieldElement([4351986304670635, 4020128726404030, 674192131526433, 1158854437106827, 6468984742885])
+        T: FieldElement([
+            4351986304670635,
```

Comment:
I see that these field elements are not aligned with the new constants. The issue is the value for x at x=3 does not provide a y coordinate that is a QR within our new curve constants. So these are based on the old constants. 

In the repo, `ristretto-skeleton` branch, there are examples of derived and calculated point additions for Extended Twisted Edwards.

--- 
My suggestion for this is: 
- This section can be deleted and you or I can use the `tools` to compute the new field elements. 
- Merge this section anyway and afterwards you or I can use the `tools` to compute the new field elements. 
- Fix this before merging but it will require can you flicking between the two codes constantly.

Reply:
You're right! 

I'll modify them for a valid variable!